src/oids.py

```diff
@@ -0,0 +1,4169 @@
+# From Peter Guttman at https://www.cs.auckland.ac.nz/~pgut001/dumpasn1.cfg
```

Comment:
I can include the program I used to process Peter's tables into a Python dictionary if you'd like.